axelrod/strategies/_filters.py

```diff
@@ -0,0 +1,179 @@
+from distutils.util import strtobool
+from collections import namedtuple
+import operator
+
+
+def passes_boolean_filter(strategy, classifier_key, value):
+    """
+    Tests whether a given strategy passes a boolean equality filter for a
+    given key in its classifier dict.
+
+    Parameters
+    ----------
+        strategy : a descendant class of axelrod.Player
+        classifier_key: string
+            Defining which entry from the strategy's classifier dict is to be
+            tested (e.g. 'stochastic' or 'inspects_source').
+        value: string or boolean
```

Comment:
It could, but it's a right nuisance. I'd have to loop through the incoming dict converting strings sometimes, but not others. I felt it was much cleaner to do it in the actual filter function.